py/visdom/server.py

```diff
@@ -665,10 +703,19 @@ def wrap_func(handler, args):
                 p['content']['data'][0]['type']))
             return
 
-        p = UpdateHandler.update(p, args)
-
-        p['contentID'] = get_rand_id()
-        broadcast(handler, p, eid)
+        p, diff_packet = UpdateHandler.update_packet(p, args)
+        if len(stringify(p)) <= len(stringify(diff_packet)):
```

Comment:
nit: simple comment to describe this block, like `# send the smaller of the patch and the updated pane`